main/apps.py

```diff
@@ -3,3 +3,6 @@
 
 class MainConfig(AppConfig):
     name = "main"
+
+    def ready(self):
+        import main.signals  # noqa
```

Comment:
Why do we go with the pattern of importing inside the function here? I would prefer to not have to `noqa` and keep imports at the top, but if there is a specific reason to go this way, I' happy to :)

Reply:
> Why do we go with the pattern of importing inside the function here?

Honestly, I don't know any other way to do it. :smiling_face_with_tear: